### ISSUE TITLE
Model blueprint has some trailing spaces that aren't "Suave"

### ISSUE THREAD

**Issue by sheriffderek** [emberjs/data#4306]
Model blueprint has some trailing spaces that aren't "Suave" https://github.com/DockYard/ember-suave

`disallowTrailingWhitespace: Illegal trailing whitespace at models/xxx.js`

I don't know if it would be any different, but I generated a resource and a model and they were the same

Removing the spaces or the line would do the trick.

Does Ember Data conform to this?

RE: https://github.com/ember-cli/ember-cli/issues/5736

**Comment by sbatson5** [comment]
I was looking at the blueprint and it looks like the newline is there because you can optionally add `attrs` when you generate your model:
`ember g model --attrs name`

Would it make sense to optionally newline it if attrs are present?  Not sure of an elegant solution.

**Comment by fivetanley** [comment]
i think you may be able to remove the trailing whitespace by wrapping that in an `if`

```javascript
<%= importStatements %>

export default Model.extend({
  <% if (attrs) { %>
    <%= attrs %>
  <% } %>
});
```

The file is located here: https://github.com/emberjs/data/blob/master/blueprints/model/files/__root__/__path__/__name__.js

**Comment by sbatson5** [comment]
I seem to always get an error when using `-%>`.  Tried it with just `<%= attrs -%>` and received an `Unexpected token` error.  Tried using it on just the `if` and received the same error.

The problem with wrapping it in an `if` is that if we do have attrs, we actually get 2 extra new lines.  Thought about doing a ternary:
`<%= typeof attrs !== 'undefined' ? attrs : '' %>`
But we would have to remove the leading spaces, otherwise we still get the empty whitespace error with Ember Suave.

My first thought was just prepending a space:

```
export default Model.extend({
<%= typeof attrs !== 'undefined' ? ' ' + attrs : '' %>
});
```
This ensures we indent the attrs but don't enforce the whitespace if there are none. However, leading spaces are added to all attrs after the first here:
https://github.com/emberjs/data/blob/master/blueprints/model/index.js#L81

Could remove it from the `index.js` file and rely on the blueprint to add the leading spaces. Doesn't seem like a good solution. I think I am missing something obvious 😛

**Comment by pangratz** [comment]
Fixed by #4380
